intake/source/cache.py

```diff
@@ -0,0 +1,228 @@
+from datetime import datetime
+from hashlib import md5
+from pathlib import Path
+
+import json
+import logging
+import os
+import shutil
+
+from intake.config import conf
+
+logger = logging.getLogger('intake')
+
+
+def disable_caching(state):
+    os.environ['INTAKE_DISABLE_CACHING'] = '1' if state else '0'
+
+class FileCache(object):
+    """
+    Provides utilities for managing cached data files.
+    """
+    blocksize = 5000000
```

Comment:
Add comment: download block size in bytes

Reply:
done